packages/account/src/providers/fuel-graphql-subscriber.ts

```diff
@@ -34,53 +34,66 @@ export class FuelGraphqlSubscriber implements AsyncIterator<unknown> {
     this.stream = response.body!.getReader();
   }
 
+  private events: Array<{ data: unknown; errors: unknown }> = [];
```

Comment:
Should `errors` be optional?

```suggestion
  private events: Array<{ data: unknown; errors?: unknown }> = [];
```